release-process.md

```diff
@@ -1,16 +1,36 @@
 # Release Process
```

Comment:
Add a note that very first deployment needs to be done by developer using cf push (and example) with appropriate service and database setup. Any following deployment is using the zero downtime push and can be done via tag.

Reply:
Done. I didn't want to go too specific as it will become outdated faster (especially with the v3-zdt-push being experimental)